apps/frontend/src/libs/types/global.type.ts

```diff
@@ -0,0 +1 @@
+export type SideBarScale = '1' | '2';
```

Comment:
I don't know what your mean.
Must be an Enum in the better case

Reply:
Oups, don't mind it. To be removed.